run_benchmarks.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -e
+
+BUNDLE="bundle100"
+while getopts b: flag
+do
+    case "${flag}" in
+        b) BUNDLE=${OPTARG};;
+    esac
+done
+
+TMPDIR=$(mktemp --directory)
+
+#echo "Currently TMP: ${TMPDIR}"
+#echo "Bundle: ${BUNDLE}"
+#cp /mnt/disk2/projects/rpmtests/bundle1000.tar.gz ${TMPDIR}
+#cp /mnt/disk2/projects/rpmtests/bundle100.tar.gz ${TMPDIR}
+
+#git clone https://github.com/artipie/rpm-adapter.git
+#cd rpm-adapter
+mvn package -Pbench
+wget https://artipie.s3.amazonaws.com/rpm-test/${BUNDLE}.tar.gz
+tar -xvzf ${BUNDLE}.tar.gz -C ${TMPDIR}
+mvn dependency:copy-dependencies
+num=$(echo ${BUNDLE} | cut -c7-11)
+echo "num: ${num}"
+echo ${TMPDIR}/bundle/${num}
```

Comment:
@milkysoft what is the meaning of these logs?

Reply:
@g4s8 , removed with latest [commit](https://github.com/artipie/rpm-adapter/pull/386/commits/8402f586a96209f7ac4ce19ab33711ee40c8bb28)